sidechain/consensus/common/src/block_import.rs

```diff
@@ -111,10 +112,14 @@ where
 		let shard = sidechain_block.shard_id();
 
 		let peeked_parentchain_header =
-			self.peek_parentchain_header(&sidechain_block, parentchain_header)?;
+			self.peek_parentchain_header(&sidechain_block, parentchain_header)
+				.unwrap_or_else(|e| {
+					warn!("Could not peek parentchain block, returning latest parentchain block ({:?})", e);
```

Comment:
It means that the sidechain block we're trying to import references a parentchain block that is older than what we currently have. This should only happen when onboarding, otherwise it's an indication of something going wrong - thus the `warn`.